data/meson.build

```diff
@@ -89,6 +89,26 @@ if build_daemon
       dynamic_options += 'RestrictAddressFamilies=AF_NETLINK AF_UNIX'
     endif
 
+    device_allows = [
+      'block-sd',
+      'char-aux',
+      'char-cpu/*',
+      'char-drm',
+      'char-gpiochip',
+      'char-hidraw',
+      'char-mei',
+      'char-nvme',
+      'char-tpm',
+      'char-usb',
+      'char-usb_device',
+    ]
+    if allow_flashrom
```

Comment:
Can you make tpm and nvme conditional too now?

Reply:
Good idea, we can do mei too.